composer.json

```diff
@@ -28,10 +29,18 @@
         }
     ],
     "autoload": {
+        "files": [
+            "drupal-phpunit-hack.php"
+        ],
```

Comment:
I can dig up the live stream where I worked on this for phpstan-drupal. It's not pretty. But I have no other idea how we could possibly mimic Drupal's runtime mutation. It creates a mutated class copy into `sites/simpletest` and essentially overwrites the default PHPUnit one by getting it autoloaded first.